src/test/java/com/microsoft/azure/kusto/kafka/connect/sink/E2ETest.java

```diff
@@ -124,7 +124,6 @@ public void testE2EJson() throws URISyntaxException, DataClientException, DataSe
         }
     }
 
-

```

Comment:
Note: I did not change this file but did format the whole project using `mvn formatter:format`